.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -92,32 +89,13 @@
           - [#1631](https://github.com/dotnet/source-build/issues/1631) tracks a fix.
         - Use <https://gist.github.com/omajid/9111a9310d452b3594b770d4ca8a3d87> as a loose baseline.
           - [#1630](https://github.com/dotnet/source-build/issues/1630) tracks a checked-in baseline.
-1.  - [ ] Add team members as reviewers to the PR. Get acceptance.
-      - Note: The PR is internal for internal builds and on GitHub for non-internal builds.
-      - [Non-Internal] Merge the PR when green and accepted.
-      - [Internal] Don't merge the PR.
 1.  - [ ] Coordinate with team and complete manual distributed smoke-testing. Suggested assignment below, but it's flexible.  Send email with direct links to artifacts from green PR.
-      - [2.1]
-        - [ ] RHEL7 and RHEL8 VMs - crummel
-        - [ ] RHEL docker - dleeapho
-        - [ ] Fedora30 - dseefeld
-        - [ ] Fedora31 - dagood
-        - [ ] Debian9 - crummel
       - [3.1]
         - [ ] RHEL 7 & 8 VM - crummel
-        - [ ] RHEL Docker - dleeapho
-        - [ ] Fedora30 - dseefeld
-        - [ ] CentOS 7 - dagood
-        - [ ] CentOS 8 - dseefeld
-        - [ ] Debian - dagood
-1.  - [ ] [Internal] Download the artifacts from the green PR validation build onto a machine with Bash.
-      - The smoke-test prereqs are platform-independent but should be taken from a non-portable build, e.g. CentOS7 Offline, *not* CentOS7 Offline Portable.
-      - We may require only a subset of the artifacts for certain releases: refer to the previous release email thread corresponding to 2.1 vs. 3.1 build.
-      - Can use <https://github.com/dotnet/source-build/blob/release/3.1/scripts/fetch-azdo-artifact-tarball.sh> as a library.
-        - E.g. download the tarballs with `download_tarball`, and fix them up with `fix_azdo_tarball` if necessary.
-1.  - [ ] [Internal] Rename tarball and smoke-test prereqs to human-friendly names. (Can use `rename_tarball_inner_dir`.)
-      - Refer to previous release's email threads for naming pattern.
-1.  - [ ] [Internal] Upload tarballs and smoke-test prereqs to blob storage.
+        - [ ] Fedora 30 - msimons
+        - [ ] CentOS 7 - msimons
+        - [ ] CentOS 8 - lbussell
+        - [ ] Debian 9 - lbussell
 1.  - [ ] [Internal] Send the tarball to partners. Include info about how certain we are that this will be the final Microsoft build.
```

Comment:
There are a few steps that should be done by a source-build team member.  These should be called out explicitly.  This is one of them.

Reply:
Added notes where I think this is needed.